assets/js/modules/optimize/datastore/index.js

```diff
@@ -56,7 +56,7 @@ baseModuleStore = ( ( { actions, selectors, ...store } ) => {
 const store = Data.combineStores(
 	baseModuleStore,
 	settings,
-	error,
+	createErrorStore(),
```

Comment:
See above, this should be in `createModuleStore` instead.

Reply:
Updated.